irohad/model/permissions.hpp

```diff
@@ -98,7 +94,7 @@ namespace iroha {
                                                      can_get_domain_signatories,
                                                      can_get_my_acc_detail};
 
-    const std::string can_grant = "CanGrant";
+    const std::string can_grant = "can_grant_";
```

Comment:
What is the purpose of end underscore?

Reply:
we has can_grant<b>_</b>can_do_smth permissions, that's the purpose